observability/distr_tracing/distr-tracing-tempo-configuring.adoc

```diff
@@ -1,38 +1,43 @@
 :_mod-docs-content-type: ASSEMBLY
 [id="distr-tracing-tempo-configuring"]
-= Configuring
+= Configuring the {DTShortName}
 include::_attributes/common-attributes.adoc[]
 :context: distr-tracing-tempo-configuring
 
 toc::[]
 
+[TIP]
+====
+For configuring the deprecated {JaegerShortName}, see xref:../../observability/distr_tracing/distr_tracing_jaeger/distr-tracing-jaeger-configuring.adoc#distr-tracing-jaeger-configuring[Configuring] in the {JaegerShortName} documentation.
```

Comment:
```suggestion
For information about configuring the deprecated {JaegerShortName}, see xref:../../observability/distr_tracing/distr_tracing_jaeger/distr-tracing-jaeger-configuring.adoc#distr-tracing-jaeger-configuring[Configuring] in the {JaegerShortName} documentation.
```

Reply:
consider not leading with a pointer to the deprecated option

Reply:
Thank you so much. Fixed.

Reply:
ack, works for me! I can see circumstances like that making sense to lead with it :)